documentation/Set-PnPTenant.md

```diff
@@ -324,7 +324,7 @@ Accept wildcard characters: False
 ```
 
 ### -DisabledWebPartIds
-Guids of out of the box modern web part id's to disallow from being added to pages and remove from pages where they already have been used. Currently only the following web parts can be disabled in such a way:
+Guid's of out of the box modern web part id's to disallow from being added to pages and remove from pages where they already have been used. Currently only the following web parts can be disabled in such a way:
```

Comment:
The previous version is the correct one.

Reply:
updated